### ISSUE TITLE
The loss of S and C optimization is different

### ISSUE THREAD

**Issue by DrewdropLife** [Never-less/SegAN#11]
The paper mentions that S and C optimization have the same loss, but in the code, dice loss is added to optimize S, and after I try to get rid of it, the effect will become very poor,  why?

**Comment by YuanXue1993** [comment]
The dice loss is used to help stabilize the adversarial training. You can try to "warm-up" the network with regular training (dice only) for several epochs and then remove the dice to use pure adversarial training, and you should be able to see the adversarial training works properly. Or you can try to use adversarial loss only from scratch, but in this case, you may have to experiment with different learning rates or even different architecture as the training can be unstable.
